Flow.Launcher/Converters/BoolToVisibilityConverter.cs

```diff
@@ -8,28 +8,14 @@ public class BoolToVisibilityConverter : IValueConverter
     {
         public object Convert(object value, System.Type targetType, object parameter, CultureInfo culture)
         {
-            if (parameter != null)
+            return value switch
             {
-                if (value is true)
-                {
-                    return Visibility.Collapsed;
-                }
-
-                else
-                {
-                    return Visibility.Visible;
-                }
-            }
-            else { 
-                if (value is true)
-                {
-                    return Visibility.Visible;
-                }
-
-                else { 
-                    return Visibility.Collapsed;
-                }
-            }
+                true when parameter is not null => Visibility.Collapsed,
```

Comment:
maybe use tuple for this?

Reply:
I completely forgot about tuples. This seems like the perfect use case for them.